ci/test/00_setup_env_arm.sh

```diff
@@ -10,7 +10,7 @@ export HOST=arm-linux-gnueabihf
 export DPKG_ADD_ARCH="armhf"
 export PACKAGES="python3-zmq g++-arm-linux-gnueabihf busybox libc6:armhf libstdc++6:armhf libfontconfig1:armhf libxcb1:armhf"
 export CONTAINER_NAME=ci_arm_linux
-export CI_IMAGE_NAME_TAG="docker.io/debian:bookworm"  # Check that https://packages.debian.org/bookworm/g++-arm-linux-gnueabihf (version 12.2, similar to guix) can cross-compile
+export CI_IMAGE_NAME_TAG="docker.io/ubuntu:noble"  # Check that https://packages.ubuntu.com/noble/g++-arm-linux-gnueabihf (version 13.3, similar to guix) can cross-compile
```

Comment:
Actually, it is 13.2:
```suggestion
export CI_IMAGE_NAME_TAG="docker.io/ubuntu:noble"  # Check that https://packages.ubuntu.com/noble/g++-arm-linux-gnueabihf (version 13.2, similar to guix) can cross-compile
```

Reply:
Are you sure:
```bash
arm-linux-gnueabihf-g++ --version
arm-linux-gnueabihf-g++ (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0
Copyright (C) 2023 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Reply:
You're right. However, the package manager reports:
```
g++-arm-linux-gnueabihf is already the newest version (4:13.2.0-7ubuntu1).
```